src/styles/blocks/menu/style.css

```diff
@@ -0,0 +1,15 @@
+@import "header.css";
+@import "menu.css";
+
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-size: 12px;
+  font-family: Roboto, sans-serif;
+
+  --color-blue: #00acdc;
+}
```

Comment:
general styles for all elements/ resetting default styles or overwritting them should be in reset.scss or normalize.scss